pkg/generate/config/field.go

```diff
@@ -284,6 +284,12 @@ type LateInitializeConfig struct {
 // read the referred 'API' resource and copy the value from 'Status.APIID' in
 // 'Integration' resource's 'APIID' field
 type ReferencesConfig struct {
+	// ServiceName mentions the AWS service name where 'Resource' exists
+	// ServiceName is the package name for AWS service package in
+	// aws-sdk-go/models/apis/<package_name>
+	// When not specified, 'ServiceName' defaults to service name of controller
+	// which contains generator.yaml
+	ServiceName string `json:"service_name,omitempty"`
```

Comment:
As per my [previous PR clarifying names](https://github.com/aws-controllers-k8s/code-generator/pull/213), this should be referred to as the `ServiceModelName`.

Reply:
I will update the comment to mention the ServicePackagePath.

This field is an abstraction that helps service teams by not having them enter complete api-group string. ACK code-generator creates the api-group string from the "ServiceName" field.

I feel it is more readable/user-friendly to leave this as simple "ServiceName" and the comment will explain that it actually is ServicePackageName from `aws-sdk-go`. Let me know your thoughts.

Even if users make a mistake like `service_name: opensearch` instead of `opensearchservice`, the error will be caught at compile time.

Reply:
I agree that it's more user friendly, sure. Please just update the code comments to make it clear for us, then 👍🏼